ttt/minimax.rb

```diff
@@ -0,0 +1,75 @@
+require_relative 'ai'
+require_relative 'symbols'
+
+class Minimax
+  @board
+  @ai_symbol
+  @current_symbol
+  @human_symbol
+  @winner
+  @symbols
```

Comment:
You can also use the trio of `attr_accessor`, `attr_reader`, `attr_writer` methods to declare variables at this level. For example, you could do:

```rb
attr_reader :board, :ai_symbol, :current_symbol, :human_symbol, :winner, :symbols
```